x-pack/plugins/spaces/public/management/edit_space/manage_space_page.tsx

```diff
@@ -154,7 +154,7 @@ export class ManageSpacePage extends Component<Props, State> {
 
         <EnabledFeatures
           space={this.state.space}
-          features={this.state.features}
+          features={this.state.features.map((f) => f.toRaw())}
```

Comment:
__nit:__ can we maybe do it once in a `componentDidMount` where we receive these features? It seems we can just switch state to `FeatureConfig` in this file too:
```ts
try {
  const featuresPromise = getFeatures().then((features) =>
    features.map((feature) => feature.toRaw())
  );

  if (spaceId) {
    await this.loadSpace(spaceId, featuresPromise);
  } else {
    this.setState({ isLoading: false, features: await featuresPromise });
  }
} catch (e) {
  .....
}
```

Reply:
fixed the problem with the types and revered the changes